### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by abnios**
seeded admin account described on the documentation

**Story card:** [5016]

## Because

After a successful docker installation of simple-server on local machine, the script running the docker will also create an admin account for the simple-server dashboard.
the newly created username and password will be
username: admin@simple.org
password: Resolve2SaveLives,
This admin account is not documented in the https://github.com/simpledotorg/simple-server#docker-compose section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,13 @@ To remove the server and clear the volumes of database data, run the command:
 ```
 docker compose down --volumes
 ```
+After a successful docker installation and running the services, The Docker will also creates a seed data for you and also will create an admin dashboard account.
+The default username and password for the dashboard account is:
+```bash
+username: admin@simple.org
+password: Resolve2SaveLives
+```
+
```
Comment (harimohanraj89) [inline README.md:76]: Can we move this full section above `To remove the server...`?

File: README.md [hunk 2]
```diff
@@ -67,6 +67,13 @@ To remove the server and clear the volumes of database data, run the command:
 ```
 docker compose down --volumes
 ```
+After a successful docker installation and running the services, The Docker will also creates a seed data for you and also will create an admin dashboard account.
```
Comment (harimohanraj89) [inline README.md:70]: This is great, just some suggestions to tighten up the wording a bit.

How about:
```
After a successful docker installation, an admin dashboard account is automatically created.
The default username...
```

**Review by harimohanraj89 — CHANGES_REQUESTED** [review]
This is great. Just a couple small suggestions from me.

**Comment by harimohanraj89** [comment]
PS, could you add the Shortcut story link in the following format to the PR description? Otherwise it doesn't pick it up automatically:

`[ch5016](https://app.shortcut.com/simpledotorg/story/5016/update-simple-server-docs-with-seeded-admin-info)`

File: README.md (hunk 2 above)
Comment (abnios) [reply README.md:70]: Okay, Will update accordingly,

File: README.md [hunk 3]
```diff
@@ -62,12 +62,23 @@ The Dockerfile and docker-compose.yml files replicate the steps detailed below f
 
 Once the Docker Compose server is running, the logs should provide the ngrok URL. For example: `SIMPLE_SERVER_HOST=91a705dde8c1.ngrok.io`. This is the value that should be used when setting up the Android app as described in the section below.
 
+After a successful docker installation, an admin dashboard account is automatically created.
+
+The default username...
```
Comment (harimohanraj89) [inline README.md:67]: Oops, I just used `...` to refer to the next sentence. We can make this `The default username and password for the dashboard account is:`

**Review by harimohanraj89 — APPROVED** [review]
Looks great 🚀

**Pull request merged**